geometry_msgs/PoseStamped current_pose
---
string area
